### ISSUE TITLE
Uglify mangling `window.__extends`

### ISSUE THREAD

**Issue by grossvogel** [valotas/preact-context#19]
Below are excerpts from the files in `node_modules/preact-context/dist/` after installing from npm. It appears that `window.__extends` is being mangled to `window.t` by the minification build step, but it probably should be excluded to function properly. As it is, if there is a global variable named `t` on the page, it will be used as `__extends` and cause `preact-context` to break.

`uglifyjs` is being passed the option `--mangle-props regex=/^_/`, which matches `__extends`, so maybe an exception should be added using `reserved`?

### context.js
```
    var __extends = (window && window.__extends) || (function () {
        var extendStatics = Object.setPrototypeOf ||
            ({ __proto__: [] } instanceof Array && function (d, b) { d.__proto__ = b; }) ||
            function (d, b) { for (var p in b) if (b.hasOwnProperty(p)) d[p] = b[p]; };
        return function (d, b) {
            extendStatics(d, b);
            function __() { this.constructor = d; }
            d.prototype = b === null ? Object.create(b) : (__.prototype = b.prototype, new __());
        };
    })();
```

### context.min.js (after formatting)
```
		e =
			(window && window.t) ||
			((i =
				Object.setPrototypeOf ||
				({ __proto__: [] } instanceof Array &&
					function(n, t) {
						n.__proto__ = t;
					}) ||
				function(n, t) {
					for (var i in t) t.hasOwnProperty(i) && (n[i] = t[i]);
				}),
			function(n, t) {
				function r() {
					this.constructor = n;
				}
				i(n, t),
					(n.prototype = null === t ? Object.create(t) : ((r.prototype = t.prototype), new r()));
			});
```

**Comment by valotas** [comment]
Thanks Jason. You are right. I'll provide a fix the following days. Sorry
for any troubles that might have costed you.

**Comment by valotas** [comment]
Just published `v1.1.1` with f4cc13a9d574d0283346d71c822b3cab1e919652 which addresses this.
